core/controllers/editor.py

```diff
@@ -607,7 +581,8 @@ def get(self, exploration_id, escaped_state_name):
         # matched to a fuzzy rule individually.
         answers = stats_services.get_top_state_rule_answers(
             exploration_id, state_name, [
-                exp_domain.DEFAULT_RULESPEC_STR, rule_domain.FUZZY_RULE_TYPE],
+                exp_domain.DEFAULT_OUTCOME_CLASSIFICATION,
+                exp_domain.TRAINING_DATA_CLASSIFICATION],
```

Comment:
These are the answers which are classified to directly match training data (semantically the same as fuzzy classified rules before).

The only TODO that's relevant is the bit about testing the handler once the storage backend does not store answers in a lossy way. This TODO will be addressable now, but can't be removed without tests being added. It will be an awesome TODO to remove, though, as it was the originator of this entire effort. :)